client/my-sites/plugins-wpcom/standard-plugins-panel.jsx

```diff
@@ -22,12 +30,11 @@ export const StandardPluginsPanel = React.createClass( {
 
 		return (
 			<div>
-				<SectionHeader label={ this.translate( 'Free Plan Plugin Suite' ) }>
+				<SectionHeader label={ this.translate( 'Standard Plugins' ) }>
```

Comment:
I am a little wary of the word "Standard" here. We want to distinguish ourselves from WordPress.org. Customer may ask - is it standard there?
Maybe something in the direction of "WordPress.COM enhancements" ? - not that copy, but that direction